applications/asset_tracker_v2/src/modules/Kconfig.data_module

```diff
@@ -63,6 +63,11 @@ config DATA_ACCELEROMETER_BUFFER_COUNT
 	range 1 100
 	default 3
 
+config DATA_IMPACT_BUFFER_COUNT
```

Comment:
It would be nice to know if heap allocation succeeds when all buffers have been filled prior to encoding. (cJSON uses dynamic allocation to allocate the JSON strings).

This will cause the application to reboot and can be a common issue in areas with poor connections. We need to be sure that with this added buffer entry we are still safe.

Reply:
Tested this with  https://github.com/maxd-nordic/sdk-nrf/pull/new/impact-detection-heap-test and resized the heap